src/Catty/Extension&Delegate&Protocol/Extensions/NotificationExtension.swift

```diff
@@ -67,4 +67,11 @@ public class NotificationName: NSObject {
     public static let settingsCrashReportingChanged = "SettingsTableViewController.crashReportingChanged"
     public static let mediaLibraryDownloadIndexFailure = "MediaLibrary.DownloadIndexFailure"
     public static let mediaLibraryDownloadDataFailure = "MediaLibrary.DownloadDataFailure"
+    public static let soundAdded = "SoundAdded"
+    public static let recordAdded = "RecordAdded"
+    public static let projectDownloaded = "ProjectDownloaded"
+    public static let hideLoadingView = "HideLoadingView"
+    public static let showSaved = "ShowSavedView"
+    public static let readyToUpload = "ReadyToUpload"
+    public static let userInfoSound = "UserInfoSound"
```

Comment:
This is not being used as a notification name, more like a key for the notification's user info. However, after looking at the removed and refactored notifications it seems like the following notifications are actually not in use anymore:

- `soundAdded` (including the user info and the never fired `soundAdded` method)
- `recordAdded`